debug/debug.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
```

Comment:
Should we add a `set -eux` here?

Reply:
Sure. The polling for the pid is a bit noisy, so I'll set `+x` around there to keep it readable.